examples/basic.rs

```diff
@@ -16,17 +16,27 @@ fn main() {
 
     let mut reader_id = event_handler.register_reader();
 
-
     // Should be empty, because reader was created after the write
-    assert_eq!(Ok([].to_vec()), event_handler.read(&mut reader_id));
+    assert_eq!(
+        Vec::<TestEvent>::default(),
```

Comment:
Why not `Vec::new()`?

Reply:
I think the rust guidelines say, use default when you can?